forge-gui/res/cardsfolder/upcoming/conclave_sledge_captain.txt

```diff
@@ -0,0 +1,14 @@
+Name:Conclave Sledge-Captain
+ManaCost:5 G
+Types:Creature Elephant Soldier
+PT:4/4
+K:Backup:1:BackupAbilities:3
```

Comment:
I think it might be better like in other cases to just put the KW on it multiple times
It shouldn't matter much but maybe something will care about the amount of it